manifests/kube/kube.json

```diff
@@ -1,40 +1,46 @@
 {
   "name": "kube",
-  "description": "A plugin to manage spin apps",
+  "description": "Commands for publishing applications to Kubernetes",
```

Comment:
Also would something along the lines of "A plugin to manage Spin apps in Kubernetes" be better?

Reply:
Or even a bit more descriptive, "A plugin to manage Spin apps as custom resources in Kubernetes"?

Reply:
I'd prefer to leave out the "custom resources" part in the description. The intent of the plugin is to manage spin apps in Kubernetes. Custom resources is how we do that today, but it doesn't always have to be that way.

Reply:
updated.